draft-thoji-scone-protocol.md

```diff
@@ -57,24 +57,34 @@ informative:
 --- abstract
 
 On-path network elements can sometimes be configured to apply rate limits to
-flows that pass them.  This document describes a method for signaling to
-endpoints that rate limiting policies are in force and  what that rate limit is.
+flows that pass them. Access networks and their attached devices may also
+experience volatile conditions that affect the sustainable network throughput. 
+
+This document describes a method for signaling to
+endpoints that rate limiting policies are in force and what that rate limit is,
+and for non-rate limiting networks to signal throughput advice based on network
+conditions.
 
 
 --- middle
 
 # Introduction
 
-Many access networks limit the maximum data rate that attached devices are able
-to attain.  This is often done without any indication to the applications
+Many access networks apply rate limits to constrain the data rate of attached
+devices. This is often done without any indication to the applications
 running on devices.  The result can be that application performance is degraded,
 as the manner in which rate limits are enforced can be incompatible with the
 rate estimation or congestion control algorithms used at endpoints.
 
 Having the network indicate what its rate limiting policy is, in a way that is
-accessible to endpoints, might allow applications to use this information when
+accessible to endpoints, allows applications to use this information when
 adapting their send rate.
 
+In addition, access networks without rate limiting policies may also provide 
+information to endpoints that can help account for connection state, such as 
+network congestion, volatile signal strength, or radio bearers with constrained
+bandwidth.
```

Comment:
I would instead say this:

```suggestion
Network elements are not limited to communicating information
about rate limiting policies.
Network elements in access networks could provide information
to endpoints that can help account for changes in network capacity
that are not suited to congestion control feedback. This might include
reduced capacity due to overuse, equipment faults, or other transient issues; 
conversely, networks might choose to signal increased availability of capacity.
```

Reply:
Thanks, that suggestion covers the motive for the PR in one paragraph :)